File: A Problems/158A - Next Round/NextRound2.scala
// http://codeforces.com/problemset/problem/158/A

import scala.io.StdIn.readLine

object NextRound2 {
    def main(args: Array[String]): Unit = {
        val Array(n, k): Array[Int] = readLine().split(" ").map(_.toInt)
        val scores: Array[Int] = readLine().split(" ").map(_.toInt)

        var advances: Int = 0
        val kth: Int = scores(k-1)

        for(score <- scores) {
            if(score >= kth && score > 0) {
                advances += 1
            }
        }

        println(advances)
    }
}

File: A Problems/158A - Next Round/NextRound2.py
# http://codeforces.com/problemset/problem/158/A

n, k = map(int, input().split(" "))
scores = list(map(int, input().split(" ")))
    
score = scores[k-1]
    
count = 0
    
for i in range(n):
    if scores[i] >= score and scores[i] > 0:
        count += 1
    
print(count)